src/Features/Core/Portable/NavigateTo/AbstractNavigateToSearchService.SearchResult.cs

```diff
@@ -90,6 +85,19 @@ private class SearchResult : INavigateToSearchResult
                     parts.Free();
                 }
             }
+
+            public string SecondarySort
```

Comment:
❓ Are we actually gaining anything with this? It appears that the secondary sort string is [always computed](https://github.com/dotnet/roslyn/blob/d4dab355b96955aca5b4b0ebf6282575fad78ba8/src/Features/Core/Portable/Remote/RemoteArguments.cs#L36) before passing the result from OOP back to VS.